### DISCUSSION TITLE
Is there any way to mask the credentials of kafka cluster in UI ?

### DISCUSSION THREAD

**Discussion by AayushSaini101** [kafbat/kafka-ui#647] [General]
<img width="1176" alt="Screenshot 2024-11-05 at 11 01 22 AM" src="https://github.com/user-attachments/assets/9b0195dd-cdd3-4bb8-939b-0fcc26cd6d14">
Hi team, we are running kafka ui in the docker and in future planning to host it so that other team can use it, but the credentials are clearly visible is there way to mask the credentials thanks

**Comment by germanosin** [comment]
Hi, @AayushSaini101 thanks your question. It's not possible right now if dynamic configs are enabled. Don't you mind to create an issue on this?

**Comment by Haarolean** [comment]
Moved to issues (#657), thanks
